src/worldcereal/job.py

```diff
@@ -203,12 +214,13 @@ def generate_map(
         JOB_OPTIONS.update(job_options)
 
     # Execute the job
+    filename_output = f"{product_type.value}_{start}_{end}"
```

Comment:
Is a separate line needed for this? A bit inconsistent with the `cropland_mask` filename construction.